Add governance.md to WG Naming (#4988)
kubernetes/community: wg-naming/governance.md

# WG Naming governance

## Uphold project values

> WG Naming upholds the Kubernetes project's [core values](./values.md).

Reviewer: ```suggestion
WG Naming upholds the Kubernetes project's [core values](/values.md).
```

> WG Naming upholds the Kubernetes project's [core values](https://github.com/kubernetes/community/blob/master/values.md).

Reviewer 2: ```suggestion
WG Naming upholds the Kubernetes project's [core values](./values.md).
```

### Inclusive is better than exclusive

> Broadly successful and useful technology requires different perspectives and skill sets which can only be heard in a welcoming and respectful environment.

> This document covers some specific ways that respect and welcome look like in WG Naming.

Reviewer: This sentence doesn't appear to be grammatically clear/correct

Author: It is correct, but I can make it clearer.

Author: Also—I am on vacation for the next two weeks, starting yesterday. I will address this feedback, and ask that you consider making any further feedback iterable by another WG Naming lead in a subsequent PR.

> Our community shows respect for the time and effort put into a discussion regardless of where a contributor is on their growth path.

Like other SIGs and working groups, WG Naming has prerequisite skills, training, and experience that are necessary for meaningful contribution. For example:

  - Education or professional experience in dismantling systemic discrimination
  - Demonstrable familiarity with current antidiscrimination resources
  - A track record of supporting inclusion in the Kubernetes project

WG Naming has many stakeholders, with many people eager to add their voices. WG Naming invites folks who are new to antiracism or inclusive language, and folks less experienced with frameworks of discrimination to participate by observing and learning. WG Naming invites contributors who meet prerequisites to contribute within the scope of WG Naming’s goals.

### Resources for antidiscrimination

https://developers.google.com/style/inclusive-documentation
https://www.antiracismresources.info/#online-resources
How to Be An Antiracist, Ibrahim X. Kendi

> So You Want to Talk About Race, Ijeoma Oluo

Reviewer 2: Can we add bullet points to these? Markdown renders this as one single line.

Reviewer 3: ```suggestion
- https://developers.google.com/style/inclusive-documentation
- https://www.antiracismresources.info/#online-resources
- _How to Be An Antiracist_, Ibrahim X. Kendi
- _So You Want to Talk About Race_, Ijeoma Oluo
```

For laziness

Reviewer 2: Thanks :pray:

### Scope of discussion

WG Naming actively moderates discussion for speech that affirms the humanity of underrepresented minorities and affirms their ongoing experience of discrimination. For example: specific suggestions for replacement terms are welcome. Disagreement over the presence and impact of racism is not welcome.

## Follow the community code of conduct

> Like all WGs and SIGs, WG Naming expects contributors to follow the [community code of conduct](CCoC). 

Reviewer 2: ```suggestion
Like all WGs and SIGs, WG Naming expects contributors to follow the [Kubernetes Code of Conduct (CoC)]. 
```

We raise this section’s topics as potential blockers to the work of WG Naming, since experience has shown they are a common and detrimental response.

> From the CCoC:

Reviewer 2: ```suggestion
From the CoC:
```

> Examples of unacceptable behavior by participants include:
>
>   - Personal attacks
>   - Trolling or insulting/derogatory comments

> **Note:** the CCoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the CCoC committee regardless of professed intent.

Reviewer 2: ```suggestion
**Note:** the CoC governs conduct, not intent. WG Naming leads may refer unacceptable conduct to the Code of Conduct Committee regardless of professed intent.
```

> [Kubernetes Code of Conduct (CoC)]: ./code-of-conduct.md

Reviewer: ```suggestion
[Kubernetes Code of Conduct (CoC)]: /code-of-conduct.md
```

> [community code of conduct]: "https://github.com/kubernetes/community/blob/master/code-of-conduct.md"

Reviewer 2: ```suggestion
[Kubernetes Code of Conduct (CoC)]: ./code-of-conduct.md
```

- Derailing behavior

    Behavior that distracts from WG Naming's purpose or attempts to redirect participants' energy into minutiae or unrelated topics or goals.

    For example: 
      - insisting that previous decisions be re-argued
      - insisting that WG Naming can take no action until all possible inequities are addressed

- [Sea lioning](https://www.forbes.com/sites/marshallshepherd/2019/03/07/sealioning-is-a-common-trolling-tactic-on-social-media-what-is-it/)

  Endless demands for evidence; refusing to accept or acknowledge peer-reviewed scientific research; insisting that one's ignorance, whether real or feigned, be satisfied by the WG group to an individual's satisfaction.

  See the original Wondermark comic: [sea lions](http://wondermark.com/1k62/)

- [Brigading](https://en.wikipedia.org/wiki/Vote_brigading)

  Attempting to overrepresent a viewpoint through participation bias

## Deviations from [sig-governance]

WG Naming leads may close or end meetings at their discretion to shut down trolling or protect the safety of community members.

WG Naming [restricts the scope of discussion](#scope-of-discussion).

> [sig-governance]: ./committee-steering/governance/sig-governance.md

Reviewer: ```suggestion
[sig-governance]: /committee-steering/governance/sig-governance.md
```

> WG Naming [restricts the scope of discussion][Scope of discussion]. 

Reviewer 2: ```suggestion
WG Naming [restricts the scope of discussion](#scope-of-discussion).
```

> [sig-governance]: "https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md"

Reviewer 2: ```suggestion
[sig-governance]: ./committee-steering/governance/sig-governance.md
```
